### PR TITLE
Fixed info logs in OVC

### PR DESCRIPTION
**Pull request by popovaan**
Root cause analysis: 

OVC prints confusing logs when verbose parameter is turned on.
log_level is printed, when OVC does not have log_level.
Memory consumption log is incorrect, it prints total consumption of script, not only convert_model().

Solution: 
Remove printing of log_level info.
Calculate memory consumption using tracemalloc, which allows to track allocated memory.

Ticket: 115349

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR - N/A
* [x]  Transformation preserves original framework node names - N/A
* [x]  Transformation preserves tensor names - N/A

Validation:
* [x]  Unit tests - N/A
* [x]  Framework operation tests - N/A
* [x]  Transformation tests - N/A

Documentation:
* [x]  Supported frameworks operations list - N/A
* [x]  Guide on how to convert the **public** model - N/A
* [x]  User guide update - N/A

### CODE REVIEW

File: tools/ovc/openvino/tools/ovc/convert_impl.py [hunk 1]
```diff
@@ -421,6 +406,9 @@ def is_verbose(argv: argparse.Namespace):
 
 
 def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
+    start_time = datetime.datetime.now()
+    tracemalloc.start()
```
Comment (mvafin) [inline tools/ovc/openvino/tools/ovc/convert_impl.py:410]: Can we measure memory only when verbose=True

Comment (popovaan) [reply tools/ovc/openvino/tools/ovc/convert_impl.py]: Yes, done.

**Review by mvafin — APPROVED** [review]

File: tools/ovc/openvino/tools/ovc/convert_impl.py [hunk 2]
```diff
@@ -500,6 +495,21 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
...
 
         send_conversion_result('success')
+
+        if is_verbose(argv):
+            elapsed_time = datetime.datetime.now() - start_time
+            print('[ SUCCESS ] Total execution time: {:.2f} seconds. '.format(elapsed_time.total_seconds()))
+
+            # Statistics on allocated memory blocks per filename and per line number
+            snapshot = tracemalloc.take_snapshot()
+            stats = snapshot.statistics('lineno')
```
Comment (rkazants) [inline tools/ovc/openvino/tools/ovc/convert_impl.py:505]: I see that `tracemalloc` does not track memory allocated not in Python: https://stackoverflow.com/questions/50148554/when-would-the-python-tracemalloc-module-allocations-statistics-not-match-whats

Can you please check you implementation on some models and compare with ps stats?
Also, why don't you use `get_traced_memory` that reports peak memory consumption?

Comment (popovaan) [reply tools/ovc/openvino/tools/ovc/convert_impl.py]: > Also, why don't you use `get_traced_memory` that reports peak memory consumption?
Yes, we can use it. Done.

> Can you please check you implementation on some models and compare with ps stats?
Here are results of comparison with top command. 

fungi_V2 (from memory)
top: 2.1 GB
tracemalloc: 327.59 MB

fungi_V2 (from file)
top: 0.1 GB
tracemalloc: 0.08 MB. 

pytorch efficientnet_b7
tracemalloc: 8.03 MB
top: 0.5 GB

Results seem to correlate, but for all cases top shows significantly larger consumption. Probably it is due to c++ memory usage, but still difference looks big.

**Review by rkazants — APPROVED** [review]

**Pull request merged**